package.json

```diff
@@ -38,7 +38,10 @@
     "watch": "./node_modules/.bin/watch 'npm run build' ./js/",
     "create-doc": "grunt jsdoc",
     "test": "http-server -p 8085",
-    "open-test": "opener http://localhost:8085/tests/index.html"
+    "open-test": "opener http://localhost:8085/tests/index.html",
+    "run-container": "docker start buildMain",
```

Comment:
Does work these Docker commands? I have installed docker now on my machine, so i can not test at the moment.

Reply:
Yeah, you only have to change the container name.

Reply:
how is your container set up can you share this?

Reply:
It's the same as what is on the readme, I only renamed it and added the npm commands because at the time I was working with multiple jsartoolkit5 repos.